SU2_CFD/src/solvers/CNEMOEulerSolver.cpp

```diff
@@ -303,7 +254,9 @@ void CNEMOEulerSolver::CommonPreprocessing(CGeometry *geometry, CSolver **solver
       unsigned long tmp = ErrorCounter;
       SU2_MPI::Allreduce(&tmp, &ErrorCounter, 1, MPI_UNSIGNED_LONG, MPI_SUM, SU2_MPI::GetComm());
       config->SetNonphysical_Points(ErrorCounter);
-
+      
+      if ((rank == MASTER_NODE) && (ErrorCounter != 0))
+        cout << "Warning. The original solution contains "<< ErrorCounter << " points that are not physical." << endl;
```

Comment:
original -> initial ?

Reply:
Yes, just copied the warning message that was there before. Will change it.